### DISCUSSION TITLE
Allowing recruitment of Age of Heroes Knalgan Alliance faction recruits?

### DISCUSSION THREAD

**Discussion by knyghtmare** [nemaara/Northern_Rebirth_Reborn#3] [Ideas]
I am at the start of coding Scenario 6 which is the more or less the midpoint of the campaign and have ran into a development choice.
Player is allowed to recruit from S3 (outlaws) and S4 (dwarves). Considering the XP modifier and rush-objectives of S4 and S5, I have noticed that the player can hardly accumulate enough experienced veterans at the start of scenario 6.

I think it would be appropriate if they are allowed to recruit level 2 units from the start of scenario 9 to the end (which is I guess only 2 scenarios but yes, it would benefit them overcome some problems in gathering XP properly)

**Comment by nemaara** [comment]
It would be better to design the scenarios in a way that allows you to get more exp. For S6, Kal Kartha's strength can be tuned higher to make up for your weaker units. It's not that great to have recruitment of lvl 2s in general I would say.

**Reply by knyghtmare** [reply]
I guess players can manage with what they have for scenarios 6 and 7. Kal Karthan forces are probably sufficient to purge drakes without saurian support.
